bcipy/helpers/triggers.py

```diff
@@ -638,3 +639,185 @@ def read_triggers(triggers_file: TextIO) -> List[Tuple[str, str, float]]:
     for _, (name, trg_type, stamp) in enumerate(records):
         corrected.append((name, trg_type, float(stamp) + offset))
     return corrected
+
+
+class TriggerType(Enum):
+    """
+    Enum for the primary types of Triggers.
+    """
+
+    NONTARGET = "nontarget"
+    TARGET = "target"
+    FIXATION = "fixation"
+    PROMPT = "first_pres_target"
```

Comment:
This may be a good time to update this to prompt outright. It will make older data less compatible but it's more clear.

Reply:
I'm guessing this is an area we'll definitely have to go back and refactor carefully?

Reply:
I think this PR will require enough refactoring and thought we can take some liberty.. let's make it "prompt"